packages/cpt-ui/components/EpsRoleSelectionPage.tsx

```diff
@@ -207,21 +243,6 @@ export default function RoleSelectionPage({ contentText }: RoleSelectionPageProp
         )
     }
 
-    // If the data is being fetched, replace the content with a spinner
```

Comment:
I think the loading spinner should be reverted, so that it shows while the user is being redirected. I think if they get a blank screen on a slow network while a redirect is loading, it'll be unclear anything is happening at all